Object Containers/src/test/TestAgeComparator.java

```diff
@@ -0,0 +1,44 @@
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class TestAgeComparator {
```

Comment:
O ipoteza mai interesanta de testat ar fi fost urmatoarea:

1. Creezi un `TreeSet` caruia ii dat comparatorul tau ca argument
2. Creezi cateva instante de `Person` cu varste diferite
3. Le adaugi in `TreeSet`
4. Verifici ca pe prima pozitie din `TreeSet` se afla persoana cu varsta cea mai mare